docker/utility/start_server.sh

```diff
@@ -25,4 +25,9 @@ echo "AUTHORIZING SFDX via JWT key auth..."
 sfdx force:auth:jwt:grant -u $SFDX_HUB_USERNAME -f /app/sfdx_hub.key -i $SFDX_CLIENT_ID --setdefaultdevhubusername
 # starting server and webpack
 yarn serve
-echo "ALL DONE!"
+
+echo "*****************************************************
+MetaCI is up and running in docker. 
+
+Go to http://localhost:8000/admin/login to log in
+******************************************************"
```

Comment:
These errors are cause the keys are already created in the db? Is this a dealbreaker/how is the best way to handle this while still having meaningful messaging?

Reply:
There are lots of reasons there could be an error.

I think I would just remove this message. when it runs successfully, `yarn serve` runs continually and does not exit, so it won't actually print the message in the situation where it would be correct.